AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -166,6 +166,7 @@ blog.livedoor.jp##.naka_rss
 blog.livedoor.jp##div[class^="footrss"]
 blog.livedoor.jp##div[class^="shitarss_"]
 blog.livedoor.jp##div[id^="shitarss_title_"]
+blog.livedoor.jp###app-follow-banner
```

Comment:
to `mobile-app_specific.txt` please, above this rule: `stolichki.ru##div[data-hint="app"]`